packages/react-components/react-carousel/stories/src/Carousel/CarouselDesignBestPractices.stories.tsx

```diff
@@ -0,0 +1,163 @@
+  subtext: {
+    marginBottom: '12px',
+    ...typographyStyles.body1,
+  },
+  container: {
+    display: 'grid',
+    gridTemplateColumns: '1fr',
+    gridTemplateRows: 'auto 1fr',
+  },
+  card: {
+    minHeight: '100px',
+  },
+  carousel: {
+    flex: 1,
+  },
+  controls: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '6px',
+
+    padding: '10px',
+  },
+  field: {
+    flex: 1,
+    gridTemplateColumns: 'minmax(100px, max-content) 1fr',
+  },
+  dropdown: {
+    maxWidth: 'max-content',
+  },
+  carouselHeader: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    gap: '10px',
+    marginBottom: '16px',
+  },
+  carouselHeaderTitle: { flex: '1', margin: '0', fontSize: '22px', fontWeight: 800 },
+  carouselNavigation: { width: 'fit-content', alignSelf: 'center', margin: '0' },
+  slider: {
+    gap: '0',
+    '& > *': {
+      margin: '0 12px',
+    },
+  },
+});
+
+const IMAGES = [
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/sea-full-img.jpg',
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/bridge-full-img.jpg',
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/park-full-img.jpg',
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/sea-full-img.jpg',
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/bridge-full-img.jpg',
+  'https://fabricweb.azureedge.net/fabric-website/assets/images/swatch-picker/park-full-img.jpg',
+];
+
+const BannerCard: React.FC<{ children: React.ReactNode; imageSrc: string; index: number }> = props => {
+  const { children, imageSrc, index } = props;
+  const classes = useClasses();
+
+  return (
+    <CarouselCard autoSize className={classes.bannerCard} aria-label={`${index + 1} of ${IMAGES.length}`}>
+      <Image fit="cover" src={imageSrc} role="presentation" className={classes.image} />
+
+      <div className={classes.cardContainer}>
+        <div className={classes.title}>{children}</div>
+        <div className={classes.subtext}>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ut enim ad minim veniam.
+        </div>
+        <div>
+          <Button appearance="primary">Call to action</Button>
+        </div>
+      </div>
+    </CarouselCard>
+  );
+};
+
+const getAnnouncement: CarouselAnnouncerFunction = (index: number, totalSlides: number, slideGroupList: number[][]) => {
+  return `Carousel slide ${index + 1} of ${totalSlides}`;
+};
+
+export const DesignBestPractices = () => {
+  const classes = useClasses();
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.card}>
+        <Carousel circular draggable announcement={getAnnouncement}>
+          <div className={classes.carouselHeader}>
+            <h1 className={classes.carouselHeaderTitle}>Carousel Title</h1>
```

Comment:
Is there a matching text component? Let's use it instead: https://react.fluentui.dev/?path=/docs/components-text--docs#presets